src/components/api-reference/api-reference.test.js

```diff
@@ -13,12 +14,14 @@ it('renders submission schema by default', () => {
   expect(component.containsMatchingElement(<Submission />)).toEqual(true);
 });
 
-it('routes to benchmarks when the hash is #historical-benchmarks', () => {
-  const component = mount(<ApiReference hash='#historical-benchmarks' />);
+it('routes to benchmarks when the activeComponent is benchmarks', () => {
```

Comment:
i left some suggestions on the tech spec, but I think it's important to preserve the hash-driven navigation if possible and to make path or hash routing possible if not.